pkg/liquidity-mining/contracts/test/MockGaugeController.sol

```diff
@@ -53,8 +53,12 @@ contract MockGaugeController is IGaugeController {
         emit NewGauge(gauge, gaugeType, 0);
     }
 
-    function add_type(string calldata, uint256) external override {
-        _numGaugeTypes += 1;
+    function add_type(string calldata, uint256 numTypes) external override {
```

Comment:
The second argument of `add_type` is not `numTypes` but rather the new type's initial weight. You can simply ignore it.

Reply:
Reverted (this was just to do a shortcut in the test, no real need for it).